joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/Defines.scala

```diff
@@ -48,7 +49,7 @@ object Defines {
       ("delete", ("delete(map[any]any, any)", "delete", voidTypeName)),
       ("imag", ("imag(ComplexType)FloatType", "imag", "FloatType")),
       ("len", ("len(any)int", "len", "int")),
-      ("make", ("make(map)map", "map", "map")),
+      ("make", ("make(map)map|make(chan)chan", "make", "map|chan")),
```

Comment:
Not the best way to handle this one with type full name as `map|chan` but for now we can live with it and rightly fix it at a later stage.